kubernetes/services/api-gateway.yaml

```diff
@@ -54,7 +61,7 @@ spec:
   selector:
     matchLabels:
       app: apimanager
-  replicas: 5
+  replicas: 10
```

Comment:
This is pretty aggressive :-) 5 is good enough.

Reply:
Sure